crates/starknet-server/tests/common/devnet.rs

```diff
@@ -72,6 +86,33 @@ impl BackgroundDevnet {
         BackgroundDevnet::spawn_with_additional_args(&[]).await
     }
 
+    /// Takes specified args and adds default values for args that are missing
```

Comment:
Otherwise I cannot use BackgroundDevnet for testing a provided `--initial-balance`. How else would I test a BackgroundDevnet with a custom port, or chain-id, or accounts number? Without this intervention, using
```rust
BackgroundDevnet::spawn_with_additional_args(&["--initial-balance", "100"])
```
would result in a command that internally uses
```
cargo run --release -- --initial-balance DEFAULT_VALUE --initial-balance 100 ...
```

In devnet-py, arg parser allowed this duplicate argument specification (it would simply use the last instance of the specifications), but here with Clap it's not allowed (it errors), so we have to make sure each CLI arg is supplied at most once.

Reply:
ok got it